tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -1418,7 +1417,7 @@ protected:
 public:
    static std::string TypeName() { return ""; }
    RCollectionField(std::string_view name, std::shared_ptr<RCollectionNTupleWriter> collectionWriter,
-                    std::unique_ptr<RNTupleModel> collectionModel);
+                    std::unique_ptr<RFieldBase> collectionMotherField);
```

Comment:
Do we (want to) expect that this is always a `RFieldZero`?

In general I was a bit confused because the rest of the RNTuple terminology seems to use "parent" as the gender-neutral term. As far as I can tell from quick searches, "mother" seems to be only used for collections - maybe we want to unify this terminology? (unless there is a reason to name it differently, of course)

Reply:
Yep, both good points! Addressed.